### PR TITLE
pin Vagrant box versions

### LINKED ISSUE #6228: integration tests: Vagrant boxes should use a specific version defined in inventory
**Is your feature request related to a problem? Please describe.**
Vagrant boxes used in CI (except `switch01` machines) always use latest boxes available in Vagrant Cloud because we upgraded to latest version before running tests.

It's a problem when you want to build new PacketFence Vagrant images on a branch in order to test changes: new images will be used when running tests on devel.

**Describe the solution you'd like**
Pin version of boxes to used in Vagrant inventory.

### PR DESCRIPTION
**Pull request by nqb**
# Description
pin Vagrant box versions for each vm used in pipeine
pin Vagrant box versions used by Packer to build PacketFence Vagrant images

# Impacts
Tests

# Issue
fixes #6228

# Delete branch after merge
YES

### CODE REVIEW

**Comment by nqb** [comment]
Looks ok according to https://gitlab.com/inverse-inc/packetfence/-/jobs/1130365456.

File: addons/vagrant/cumulus/Vagrantfile [hunk 1]
```diff
@@ -249,6 +249,7 @@ vagrant_interface_rule
     device.vm.hostname = "node01" 
     
     device.vm.box = "debian/buster64"
+    device.vm.box_version = "10.20210228.1"
```
Comment (JeGoi) [inline addons/vagrant/cumulus/Vagrantfile]: Is there a reason that this one is not in the hosts file?

Comment (JeGoi) [reply addons/vagrant/cumulus/Vagrantfile]: Can we also move the cumulus vm.box_version in the hosts file?

**Review by JeGoi — COMMENTED** [review]
Two small questions then it looks perfect for me.

File: addons/vagrant/cumulus/Vagrantfile (hunk 1 above)
Comment (nqb) [reply addons/vagrant/cumulus/Vagrantfile]: Done.

Comment (JeGoi) [reply addons/vagrant/cumulus/Vagrantfile]: Cool, then let's merge it.

**Pull request merged**